src/core/MenuSelect/index.stories.tsx

```diff
@@ -0,0 +1,302 @@
+import ButtonBase from "@material-ui/core/ButtonBase";
+import Popper from "@material-ui/core/Popper";
+import { makeStyles } from "@material-ui/core/styles";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { AutocompleteCloseReason } from "@material-ui/lab";
+import { Args, Story } from "@storybook/react";
+import React from "react";
+import Chip from "../Chip";
+import MenuSelect, { DefaultMenuSelectOption } from "./index";
+
+const Demo = (props: Args): JSX.Element => {
```

Comment:
This Demo component should have the structure pretty close to what we need to build ComplexFilterComopnent in [this ticket](https://app.clubhouse.io/sci-design-system/story/152399/complex-filter-integration). And We'd still need to add more props to allow controlled/uncontrolled component behavior, etc..

So @mmmmmmaya please feel free to use this demo as the base for the implementation!

The only change I'd do is to avoid using `const classes = useStyles();`, since Material UI has moved away from that way of styling and prefer using `styled()`. Please kindly check [this doc](https://github.com/chanzuckerberg/sci-components/blob/main/docs/contribution.md#styling) for the styling patterns we identified so far. And if those patterns don't work for you, please reach out and we'll solve it together 🤩 💪